### PR TITLE
go.mod: bump dependencies

### PR DESCRIPTION
**Pull request by erikgrinaker**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -389,12 +404,16 @@ github.com/stretchr/testify v1.6.1/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/
...
 github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d h1:gZZadD8H+fF+n9CmNhYL1Y0dJB+kLOmKd7FbPJLeGHs=
 github.com/syndtr/goleveldb v1.0.1-0.20190923125748-758128399b1d/go.mod h1:9OrXJhf154huy1nPWmuSrkgjPUtUNhA+Zmy+6AESzuA=
+github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca h1:Ld/zXl5t4+D69SiV4JoN7kkfvJdOWlPpfxrzxpLMoUk=
+github.com/syndtr/goleveldb v1.0.1-0.20200815110645-5c35d600f0ca/go.mod h1:u2MKkTVTVJWe5D1rCvame8WqhBd88EuIwODJZ1VHCPM=
 github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c h1:g+WoO5jjkqGAzHWCjJB1zZfXPIAaDpzXIEJ0eS6B5Ok=
 github.com/tecbot/gorocksdb v0.0.0-20191217155057-f0fad39f321c/go.mod h1:ahpPrc7HpcfEWDQRZEmnXMzHY03mLDYMCxeDzy46i+8=
 github.com/tendermint/tendermint v0.34.0-rc3 h1:d7Fsd5rdbxq4GmJ0kRfx7l7LesQM7e70f0ytWLTQ/Go=
 github.com/tendermint/tendermint v0.34.0-rc3/go.mod h1:BoHcEpjfpBHc1Be7RQz3AHaXFNObcDG7SNHCev6Or4g=
 github.com/tendermint/tm-db v0.6.1 h1:w3X87itMPXopcRPlFiqspEKhw4FXihPk2rnFFkP0zGk=
 github.com/tendermint/tm-db v0.6.1/go.mod h1:m3x9kRP4UFd7JODJL0yBAZqE7wTw+S37uAE90cTx7OA=
```
Comment (marbar3778) [inline go.sum:414]: go mod tidy?

Comment (erikgrinaker) [reply go.sum:414]: This is tidied.

Comment (marbar3778) [reply go.sum:414]: ah odd it didn't remove the previous versions

**Review by marbar3778 — APPROVED** [review]

File: go.sum (hunk 1 above)
Comment (erikgrinaker) [reply go.sum:414]: Probably used in some other dependency, maybe tm-db.

Comment (marbar3778) [reply go.sum:414]: yes, but wouldn't 0.6.1 of tm-db then be removed? not a biggie but my understanding of go mod tidy may be wrong

Comment (erikgrinaker) [reply go.sum:414]: I think because tm-db 0.6.1 is a dependency of tendermint 0.34-0-rc3.

Comment (marbar3778) [reply go.sum:414]: welp, yes forgot this repo imports Tendermint. 😢

Comment (erikgrinaker) [reply go.sum:414]: We may want to split the crypto/Merkle stuff out to a separate library.

Comment (marbar3778) [reply go.sum:414]: YAY, this is what I was trying to do with tm-db, before it was named tm-db. It was initially tm-cmn and was meant to house the cross repo deps.

**Pull request merged**